Installation of v0.14.1 from git dosn't work

```
$ cargo install --git https://github.com/chevdor/subwasm --tag v0.14.1
    Updating git repository `https://github.com/chevdor/subwasm`
  Installing subwasm v0.14.1 (https://github.com/chevdor/subwasm?tag=v0.14.1#e3ac02f4)
    Updating crates.io index
    Updating git repository `https://github.com/paritytech/jsonrpsee`
    Updating git repository `https://github.com/paritytech/substrate/`
error: failed to compile `subwasm v0.14.1 (https://github.com/chevdor/subwasm?tag=v0.14.1#e3ac02f4)`, intermediate artifacts can be found at `/tmp/cargo-install8LK6lb`

Caused by:
  failed to select a version for the requirement `jsonrpsee = "^0.3"`
  candidate versions found which didn't match: 0.4.1
  location searched: Git repository https://github.com/paritytech/jsonrpsee
  required by package `wasm-loader v0.14.1 (/home/elois/.cargo/git/checkouts/subwasm-3353e42aad30a3df/e3ac02f/libs/wasm-loader)`
      ... which is depended on by `subwasm v0.14.1 (/home/elois/.cargo/git/checkouts/subwasm-3353e42aad30a3df/e3ac02f/cli)`
```

Sorry @chevdor , here are the logs:
```
└────╼ cargo build --release -p subwasm --locked
   Compiling zstd-safe v4.1.1+zstd.1.5.0
   Compiling libc v0.2.103
   Compiling serde v1.0.130
   Compiling futures-executor v0.3.17
   Compiling tokio-util v0.6.8
   Compiling clap v3.0.0-beta.4
   Compiling jsonrpsee-proc-macros v0.4.1 (https://github.com/paritytech/jsonrpsee#e734afe2)
error[E0658]: arbitrary expressions in key-value attributes are unstable
 --> /home/alan/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.0.0-beta.4/src/lib.rs:8:10
  |
8 | #![doc = include_str!("../README.md")]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: see issue #78835 <https://github.com/rust-lang/rust/issues/78835> for more information

   Compiling futures v0.3.17
error[E0658]: use of unstable library feature 'array_map'
   --> /home/alan/.cargo/git/checkouts/jsonrpsee-5ae67701a3cedc92/e734afe/proc-macros/src/attributes.rs:111:54
    |
111 |         let mut result: [Result<Argument, _>; N] = allowed.map(|name| Err(MissingArgument(self.path.span(), name)));
    |                                                            ^^^
    |
    = note: see issue #75243 <https://github.com/rust-lang/rust/issues/75243> for more information

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
   Compiling soketto v0.7.0
error: could not compile `jsonrpsee-proc-macros`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: build failed
```

--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ Any node can be queried to provide its current metadata. This can be displayed i
 
 ### Using Cargo
 
-    cargo install --git https://github.com/chevdor/subwasm --tag v0.14.1
+    cargo install --git https://github.com/chevdor/subwasm --tag v0.14.1 --locked
 
 ### Homebrew
 

--- a/README_src.adoc
+++ b/README_src.adoc
@@ -45,7 +45,7 @@ image::scripts/demos/gif/demo-diff.gif[title="subwasm diff"]
 
 [subs="attributes+"]
 ----
-cargo install --git https://github.com/chevdor/subwasm --tag {version}
+cargo install --locked --git https://github.com/chevdor/subwasm --tag {version}
 ----
 
 === Homebrew

--- a/libs/substrate-differ/src/call_wrapper.rs
+++ b/libs/substrate-differ/src/call_wrapper.rs
@@ -12,7 +12,7 @@ impl<'a, K: Display, V: Display + 'a> Display for CallWrapper<'a, K, V> {
 					res.push_str(&format!("{}.", key));
 				}
 
-				write!(f, "🛠  {:<50} {:>20} --> {}", res, v_before.to_string(), v_after.to_string())
+				write!(f, "🛠  {:<50} {:>20} --> {}", res, v_before, v_after)
 			}
 			ChangeType::Removed(keys, val) => {
 				let mut res = String::new();

--- a/libs/substrate-differ/src/differs/raw_differ.rs
+++ b/libs/substrate-differ/src/differs/raw_differ.rs
@@ -59,7 +59,7 @@ impl<'a, T: Serialize> RawDiffer<'a, T> {
 					{
 						skip_counter.bytes += 1;
 					} else {
-						println!("{}", wrapped_call.to_string())
+						println!("{}", wrapped_call)
 					}
 
 					// if !k.contains(&doc) || k.contains(&doc) && !options.skip_doc {

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -1,0 +1,4 @@
+[toolchain]
+channel = "stable"
+components = [ "rustfmt", "clippy" ]
+profile = "minimal"

--- a/templates/release.md
+++ b/templates/release.md
@@ -8,7 +8,7 @@ Download the binary for your OS from below:
 ### From source
 
 ```
-cargo install --git https://github.com/chevdor/subwasm
+cargo install --locked --git https://github.com/chevdor/subwasm
 ```
 
 ### Linux

